### DISCUSSION TITLE
How to access TRNG?

### DISCUSSION THREAD

**Discussion by gertvdijk** [solokeys/solo2-cli#92] [General]
For Solo 1, one can use the TRNG on the device as random source on the host computer using [`solo key rng`](https://docs.solokeys.dev/solo-extras/).

A similar command for the Solo 2 CLI appears to be missing. Is it not available on the hardware or is it just omitted in the CLI tooling? If it's the latter, can this be added? Thanks.

<sup>The page (for Solo 1) does not indicate it's for Solo 1 only, and I was expecting to use the TRNG on my new Solo 2... 😬</sup>

**Comment by coelner** [comment]
https://github.com/solokeys/solo2-cli/discussions/87
